resources/codegen_inputs/fvar.rs

```diff
@@ -0,0 +1,56 @@
+#![parse_module(read_fonts::tables::fvar)]
+
+extern record InstanceRecord;
+
+/// The [fvar (Font Variations)](https://docs.microsoft.com/en-us/typography/opentype/spec/fvar) table
+#[tag = "fvar"]
+table Fvar {
+    /// Major version number of the font variations table — set to 1.
+    /// Minor version number of the font variations table — set to 0.
+    version: MajorMinor,
+    /// Offset in bytes from the beginning of the table to the start of the VariationAxisRecord array. The
+    /// InstanceRecord array directly follows.
+    #[read_offset_with($axis_count, $instance_count, $instance_size)]
+    axis_instance_arrays_offset: Offset16<AxisInstanceArrays>,
+    /// This field is permanently reserved. Set to 2.
+    #[skip_getter]
+    #[compile(2)]
+    _reserved: u16,
+    /// The number of variation axes in the font (the number of records in the axes array).
+    axis_count: u16,
+    /// The size in bytes of each VariationAxisRecord — set to 20 (0x0014) for this version.
+    #[compile(20)]
+    axis_size: u16,
```

Comment:
in theory, I imagine we want to be using this value to determine the stride length during parsing, or else we risk incorrect results with some future spec bump. There's a similar issue in STAT, where it is also not resolved; I'm okay with leaving it as-is for now, but we should definitely open an issue.

I have a half-written patch for STAT somewhere, so hopefully the pattern I use there will also be appropriate here.

aside: the idea of a record that may change size in the future is nasty, I feel like this should *really* just be a table with a version number. :/

Reply:
I agree, and I think the spec is completely broken here. I honestly don't see how they can change either of the sizes in this table without breaking just about every implementation in the wild. 

I'll keep this as-is and let you carry over whatever pattern you're considering for `STAT` when compilation comes up for this table.